### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Lumiahna**
Update French translation

### CODE REVIEW

File: i18n/fr.json [hunk 1]
```diff
@@ -27,15 +27,15 @@
   "app.config.ok": "OK",
   "app.config.reset": "Effacer les paramètres et les données",
   "app.config.selectAccount": "Choisissez votre compte WoW",
-  "app.config.showallauras": "Show auras without updates",
+  "app.config.showallauras": "Montrer les auras non mises à jour",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.config.showallauras": "Montrer les auras sans mise à jour",
```

File: i18n/fr.json [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
-  "app.config.autoupdater.beta": "Use Companion Beta channel",
-  "app.config.autoupdater.check": "Check for COmpanion Updates",
-  "app.config.backup.activate": "Activate",
-  "app.config.backup.backupfolder": "Backup Folder",
-  "app.config.backup.dedicatedsize": "Dedicated size",
+  "app.config.autoupdater": "Mise à jour automatique",
+  "app.config.autoupdater.autoupdate": "Installer automatiquement les mises à jour de companion",
+  "app.config.autoupdater.beta": "Utiliser la beta de companion",
+  "app.config.autoupdater.check": "Rechercher des mises à jour de companion",
+  "app.config.backup.activate": "Activé",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.config.backup.activate": "Activer",
```

File: i18n/fr.json [hunk 3]
```diff
@@ -27,15 +27,15 @@
...
+  "app.config.wagoAccount": "Compte Wago (optionnel)",
+  "app.config.wagoApiKey": "Définir la clef API Wago (optionnel)",
   "app.config.wagoSettings": "Paramètres de Wago",
-  "app.dropdown.placeholder": "Select",
-  "app.fileselect.backupfolder": "Backup Folder",
+  "app.dropdown.placeholder": "Selectionner",
+  "app.fileselect.backupfolder": "Dossier de sauvegarde",
   "app.fileselect.wowfolder": "Dossier de World of Warcraft",
-  "app.footer.getweakauras": "Get WeakAuras!",
+  "app.footer.getweakauras": "Récupérer des WeakAuras!",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.footer.getweakauras": "Téléchargez WeakAuras!",
```

File: i18n/fr.json [hunk 4]
```diff
@@ -76,38 +76,38 @@
...
   "app.main.installTotal": "Tout est déjà à jour! | 1 mise à jour prête à être installé en jeux | {n} mises à jours prêtes à être installé en jeux",
-  "app.main.installUpdate": "Install client update",
+  "app.main.installUpdate": "Installer la mise à jour du client",
   "app.main.later": "Plus tard",
   "app.main.needreload": "Rechargez l'interface de World of Warcraft pour voir les mises à jour dans les options de WeakAuras",
   "app.main.needreloadstash": "Rechargez l'interface de World of Warcraft pour recevoir les auras",
   "app.main.needrestart": "Relancez World of Warcraft pour voir les mises à jours dans les options de WeakAuras",
   "app.main.nothingToFetch": "Pas de mise à jour disponible",
-  "app.main.settingswiped": "Companion's settings have been reset!",
+  "app.main.settingswiped": "Les paramètres compagnon ont ete remises à zéro!",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.main.settingswiped": "Les paramètres de Compagnon ont été remis à zéro!",
```

File: i18n/fr.json [hunk 5]
```diff
@@ -76,38 +76,38 @@
...
   "app.main.stringReceiveError-1": "Problème lors de la réception des données de {aura}",
   "app.main.stringReceiveError-2": "http code: {status}",
   "app.main.updatedownload": "Mise à jour du Client téléchargée.",
   "app.main.updateerror": "Error pendant la mise à jour. {arg}",
-  "app.main.updatefound": "Companion Update available",
+  "app.main.updatefound": "Mise à jour companion disponible",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.main.updatefound": "Mise à jour de Companion disponible",
```

File: i18n/fr.json [hunk 6]
```diff
@@ -9,15 +9,15 @@
...
-  "app.config.autoupdater.autoupdate": "Install Companion updates automatically",
-  "app.config.autoupdater.beta": "Use Companion Beta channel",
-  "app.config.autoupdater.check": "Check for COmpanion Updates",
-  "app.config.backup.activate": "Activate",
-  "app.config.backup.backupfolder": "Backup Folder",
-  "app.config.backup.dedicatedsize": "Dedicated size",
+  "app.config.autoupdater": "Mise à jour automatique",
+  "app.config.autoupdater.autoupdate": "Installer automatiquement les mises à jour de companion",
+  "app.config.autoupdater.beta": "Utiliser la beta de companion",
+  "app.config.autoupdater.check": "Rechercher des mises à jour de companion",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.config.autoupdater.check": "Rechercher des mises à jour de Companion",
```

File: i18n/fr.json [hunk 7]
```diff
@@ -9,15 +9,15 @@
...
   "app.config.autoStart": "Démarrer le client avec votre ordinateur",
-  "app.config.autoupdater": "Auto-Update",
-  "app.config.autoupdater.autoupdate": "Install Companion updates automatically",
-  "app.config.autoupdater.beta": "Use Companion Beta channel",
-  "app.config.autoupdater.check": "Check for COmpanion Updates",
-  "app.config.backup.activate": "Activate",
-  "app.config.backup.backupfolder": "Backup Folder",
-  "app.config.backup.dedicatedsize": "Dedicated size",
+  "app.config.autoupdater": "Mise à jour automatique",
+  "app.config.autoupdater.autoupdate": "Installer automatiquement les mises à jour de companion",
```
Comment (mrbuds) [inline i18n/fr.json]:
```suggestion
  "app.config.autoupdater.autoupdate": "Installer automatiquement les mises à jour de Companion",
```

**Comment by Lumiahna** [comment]
Looks good for me

**Comment by mrbuds** [comment]
You have to commit all suggestions so i can merge the PR, i don't have permissions on your repository.

**Pull request merged**